src/EventListener/ControllerListener.php

```diff
@@ -60,6 +60,20 @@ public function onKernelController(KernelEvent $event)
         $classConfigurations = $this->getConfigurations($this->reader->getClassAnnotations($object));
         $methodConfigurations = $this->getConfigurations($this->reader->getMethodAnnotations($method));
 
+        if (method_exists($method, 'getAttributes')) {
+            $classAttributes = $object->getAttributes();
+            array_walk($classAttributes, function (\ReflectionAttribute &$value) {
+                $value = $value->newInstance();
+            });
```

Comment:
You are creating new instances of _every_ attribute that is attached to the class. This is something you should never do. The class might have all kinds of attributes, not only those that come from this bundle. Your code will break if there are attributes attached that cannot be instantiated, e.g. if the referenced attribute class is not defined.

Please only instantiate attributes that are relevant for you!